### PR TITLE
Updated User Guide for Amazon SNS 4.4.0

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: modules/ROOT/pages/amazon/amazon-sns-connector-upgrade-migrate.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+|===
+
+See the xref:release-notes::connector/amazon-sns-connector-release-notes-mule-4.adoc[Amazon SNS Connector Release Notes] for updated compatibility information for patch updates.
+
+== Upgrade Prerequisites
+
+Before you perform the upgrade, create a backup of your files, data, and configuration in case you need to restore to the previous version.
+
+== See Also
+https://help.mulesoft.com[MuleSoft Help Center]
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/amazon/amazon-sns-connector-upgrade-migrate.adoc:41]:
```suggestion
* https://forums.mulesoft.com[MuleSoft Forum]
```

File: modules/ROOT/pages/amazon/amazon-sns-connector.adoc [hunk 2]
```diff
@@ -41,12 +41,8 @@ Amazon SNS connector is compatible with:
 
 == Audience
 
-* Starting user
-+
-To create your Mule app, see xref:amazon/amazon-sns-connector-design-center.adoc[Design Center Configuration] or xref:amazon/amazon-sns-connector-studio.adoc[Studio Configuration]. The xref:amazon/amazon-sns-connector-examples.adoc[Examples] topic provides one or more use cases for using this connector.
-* Power user
-+
-Read the xref:amazon/amazon-sns-connector-xml-maven.adoc[XML and Maven Support]
+* Starting user -- To create your Mule app, see xref:amazon/amazon-sns-connector-design-center.adoc[Design Center Configuration] or xref:amazon/amazon-sns-connector-studio.adoc[Studio Configuration]. The xref:amazon/amazon-sns-connector-examples.adoc[Examples] topic provides one or more use cases for using this connector.
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/amazon/amazon-sns-connector.adoc]:
```suggestion
* Starting user - To create your Mule app, see xref:amazon/amazon-sns-connector-design-center.adoc[Design Center Configuration] or xref:amazon/amazon-sns-connector-studio.adoc[Studio Configuration]. The xref:amazon/amazon-sns-connector-examples.adoc[Examples] topic provides one or more use cases for using this connector.
```

**Review by wwarner-salesforce — APPROVED** [review]

File: modules/ROOT/pages/amazon/amazon-sns-connector-upgrade-migrate.adoc (hunk 1 above)
Comment (dhirsch1) [reply modules/ROOT/pages/amazon/amazon-sns-connector-upgrade-migrate.adoc:41]: Actually, these should all be references to the Help Center, which I think is replacing the Forum. I will fix the others.

Comment (wwarner-salesforce) [reply modules/ROOT/pages/amazon/amazon-sns-connector-upgrade-migrate.adoc:41]: Oops! Thanks for catching that, and for fixing the others.

File: modules/ROOT/pages/amazon/amazon-sns-connector.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/amazon/amazon-sns-connector.adoc]: I may have also made the wrong correction here. "Starting user" has two dashes, and "Power user" has one dash. I guessed that this one should change but maybe it's the other that should change.

**Comment by dhirsch1** [comment]
This PR included the Reference Guide topic, so put the changes in a new PR.

**Pull request closed without merging**